gitbootcamp.rst

```diff
@@ -551,3 +551,36 @@ To edit an open pull request that targets ``main``:
       git push git@github.com:<contributor>/cpython <pr_number>:<branch_name>
 
 5. Optionally, :ref:`delete the PR branch <deleting_branches>`.
+
+
+GitHub CLI
+----------
+
+GitHub CLI is a CLI (Command Line Interface) to work with GitHub resources.
+You can create new pull request, create new release, set alias command,
+check PR status, and etc, using this CLI.
+
+You can install GitHub CLI `by following this installation instructions.
+<https://github.com/cli/cli#installation`__
+
+Example of useful commands:
+
+#. Login::
+
+      gh auth login
+
+#. Create a PR::
+
+      gh pr create
+
+#. Checkout others PR::
+
+      gh pr checkout <pr-id>
+
+#. Set ``ssh`` at the git protocol::
+
+      gh config set git_protocol ssh
+
+#. Set the browser::
```

Comment:
I think this still creates a numbered list.

You should try to build the docs locally to verify that the output looks ok.
See https://devguide.python.org/docquality/#developer-s-guide-workflow

Reply:
Thanks for pointing this @ezio-melotti :rocket: